src/networking/VoiceWebSocket.ts

```diff
@@ -32,6 +32,7 @@ export class VoiceWebSocket extends WebSocket {
 	/**
 	 * Creates a new VoiceWebSocket
 	 * @param address The address to connect to
+	 * @param debug Whether or not we should print debug logs
```

Comment:
This does not print anything, it just enables the emission of debug events.
```suggestion
	 * @param debug Whether or not debug logs should be emitted
```